util/src/main/java/io/kubernetes/client/util/Yaml.java

```diff
@@ -350,7 +350,11 @@ private IntOrString constructIntOrString(ScalarNode node) {
     }
 
     private Object constructDateTime(ScalarNode node) {
-      return new DateTime(((ScalarNode) node).getValue(), DateTimeZone.UTC);
+    	if(node.getValue() == null || "null".equalsIgnoreCase(node.getValue())) {
```

Comment:
please run `mvn fmt:fmt` to properly format this.

(side note, CI/CD isn't working for this repository anymore I think the travis permissions expired or something, we should switch to github actions)

Reply:
review comments are incorporated and pushed to existing PR.
"switch to github actions" --> any further actions required from my side?

Reply:
No, nothing on your side, but I'm going to block merge until we get CI/CD reactivated.